internal/manifest/list.go

```diff
@@ -65,10 +65,12 @@ type List interface {
 // ListUpdate includes the fields which a List's UpdateInstances() method will modify.
 // This is publicly visible as c/image/manifest.ListUpdate.
 type ListUpdate struct {
-	Digest    digest.Digest
-	Size      int64
-	MediaType string
-	Platform  *imgspecv1.Platform // read-only field: may be set by Instance(), ignored by UpdateInstance()
+	Digest                    digest.Digest
+	Size                      int64
+	MediaType                 string
+	Platform                  *imgspecv1.Platform // read-only field: may be set by Instance(), ignored by UpdateInstance()
+	Annotations               map[string]string   // read-only field: may be set by Instance(), ignored by UpdateInstance()
+	CompressionAlgorithmNames []string            // read-only field: may be set by Instance(), ignored by UpdateInstance()
```

Comment:
The schema2 implementation should return `[gzip]` here, so that a copy of a schema2 multi-arch image, converting to OCI and adding the Zstd variants, works consistently with a copy of a gzip OCI image.

Reply:
Done.